dataset/models/tf/linknet.py

```diff
@@ -124,7 +124,7 @@ def decoder_block(cls, inputs, filters, name, **kwargs):
         """
         upsample_args = cls.pop('upsample', kwargs)
 
-        num_filters = inputs.get_shape()[-1].value // 4
+        num_filters = cls.get_num_channels(inputs, kwargs.get('data_format')) // 4
```

Comment:
why `get_`? there is a `num_channels` method.

Reply:
Static 'num_channels' exists in 'data_format' branch but in 'nn' num_channels isn't static method.